storage.js

```diff
@@ -32,21 +32,20 @@ module.exports = function (generate) {
   function constructKeys(keys, legacy) {
     if(!keys) throw new Error('*must* pass in keys')
 
-    return [
-    '# this is your SECRET name.',
-    '# this name gives you magical powers.',
-    '# with it you can mark your messages so that your friends can verify',
-    '# that they really did come from you.',
-    '#',
-    '# if any one learns this name, they can use it to destroy your identity',
-    '# NEVER show this to anyone!!!',
-    '',
-    legacy ? keys.private : JSON.stringify(keys, null, 2),
-    '',
-    '# WARNING! It\'s vital that you DO NOT edit OR share your secret name',
-    '# instead, share your public name',
-    '# your public name: ' + keys.id
-    ].join('\n')
+    return `# WARNING: Do not edit, share, or use this file on two devices at once.
+#
+# This is your SECRET, it gives you magical powers. With your secret you can
+# sign your messages so that your friends can verify that the messages came
+# rom you. If anyone learns your secret, they can use it to impersonate you.
+#
+# If you use this secret on more than one device you will create a fork and
+# your friends will stop replicating your content.
+#
+${legacy ? keys.private : JSON.stringify(keys, null, 2)}
```

Comment:
This looks good to me. The only thing I'm not sure about is that you've change the format from  `\n\nKey\n\n` to `#\nKey\n#` ... or something.

Just wanted to check that you checkout how this file is parsed.

Reply:
recommend you run this to see if it explodes, or write tests.
This is ready to merge otherwise

This looks looks like a great change, good thinking!

Reply:
Thanks! The regex removes [these lines](https://regex101.com/r/JS8Rci/1), and since it parses the rest as JSON the empty lines can either stay or go. We can re-add them if you think it looks better. :man_shrugging: 

Good idea on making sure we have tests. The tests are [here](https://github.com/ssbc/ssb-keys/blob/588846380f0982c01de884b1c2409ee4f3c51ae0/test/index.js#L6-L19), and I manually confirmed that the test fails if you break the file. This commit *should* be good to go, as far as tests are concerned.

Reply:
Cool. Sounds like it's ready to go then @christianbundy 
MERGE AWAY!